Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -37,14 +37,24 @@ static std::unordered_map<std::string, std::string> s_textureMap;
 static std::unordered_map<std::string, std::shared_ptr<HiresTexture>> s_textureCache;
 static std::mutex s_textureCacheMutex;
 static std::mutex s_textureCacheAquireMutex;  // for high priority access
-static Common::Flag s_textureCacheAbortLoading;
+static Common::Flag s_texturePrefetcher_AbortLoading;
 static bool s_check_native_format;
 static bool s_check_new_format;
 
 static std::thread s_prefetcher;
 
 static const std::string s_format_prefix = "tex1_";
 
+// Prefetching counters
+static u32 FileType_Counter_DDS_Amount;
```

Comment:
I feel like this could be done a bit more elegantly. Perhaps an `unordered_map` with an enum, so all of these variables (with the exception of the last one) can be stored in one object? A struct could also suffice.